config/routes.rb

```diff
@@ -1,81 +1,32 @@
 # frozen_string_literal: true
 
 # rubocop:disable Metrics/BlockLength
-# TODO: see https://github.com/SUSE/Portus/issues/1469
 Rails.application.routes.draw do
-  resources :errors, only: [:show]
-  resources :teams, only: %i[index show update] do
-    member do
-      get "typeahead/:query" => "teams#typeahead", :defaults => { format: "json" }
-    end
-  end
-  get "/teams/typeahead/:query" => "teams#all_with_query", :defaults => { format: "json" }
 
+	def draw(routes_module)
+    instance_eval(File.read(Rails.root.join("config/routes/#{routes_module}.rb")))
+	end
+  resources :errors, only: [:show]
+	draw :teams
   resources :help, only: [:index]
 
   resources :team_users, only: %i[create destroy update]
-  resources :namespaces, only: %i[index show update] do
-    put "change_visibility", on: :member
-    resources :webhooks do
-      resources :headers, only: %i[create destroy], controller: :webhook_headers
-      resources :deliveries, only: [:update], controller: :webhook_deliveries
-      member do
-        put "toggle_enabled"
-      end
-    end
-  end
-  get "namespaces/typeahead/:query" => "namespaces#typeahead", :defaults => { format: "json" }
-
-  resources :repositories, only: %i[index show destroy] do
-    post :toggle_star, on: :member
-    resources :comments, only: %i[create destroy]
-  end
-
+	draw :namespaces
+	draw :repositories
   resources :tags, only: %i[show destroy]
-
   resources :application_tokens, only: %i[create destroy]
-
-  devise_for :users, controllers: { registrations:      "auth/registrations",
-                                    sessions:           "auth/sessions",
-                                    passwords:          "passwords",
-                                    omniauth_callbacks: "auth/omniauth_callbacks" }
+	draw :user
   resource :dashboard, only: [:index]
   resources :search, only: [:index]
   resources :explore, only: [:index]
```

Comment:
Let's group this three into a `root` file. It could also contain the `help` resource (it should be harmless to change the order of this one)

Reply:
done.